settings.yml

```diff
@@ -0,0 +1,6 @@
+---
+username: "admin"
```

Comment:
This file could live in the manifest repository. I'll change it in the future since we have a pending reborganization happening within it.